mesonbuild/interpreter/primitives/string.py

```diff
@@ -120,9 +121,12 @@ def split_method(self, args: T.Tuple[T.Optional[str]], kwargs: TYPE_kwargs) -> T
     @noKwargs
     @typed_pos_args('str.strip', optargs=[str])
     def strip_method(self, args: T.Tuple[T.Optional[str]], kwargs: TYPE_kwargs) -> str:
+        if args[0]:
+            FeatureNew.single_use('str.strip with characters strip string-list', '0.43.0', self.subproject)
```

Comment:
As a user I don't think I'd really understand what this message is trying to convey. What about "str.strip with positional arguments"?

Reply:
As a developer, I don't understand it neither :D

Reply:
As a developer I see "str.strip with characters", ignore the second half, and assume "characters" means "something as opposed to nothing". I can do that since I already expect that from the "str.strip" part. :D

Reply:
I think it's unlikely we'll add another one, but if we did then the FeatureNew could be changed to state "with optional first/second positional parameter" so I don't think it's *that* confusing. :)

Reply:
@xclaesse I am still not entirely sure what to think of this wording, so what do you think about it?

The PR currently uses:
```diff
-str.strip with characters strip string-list
+str.strip with stripped characters string
```

Reply:
I would have said simply `str.strip with a positional argument`.

Reply:
You don't really need to explain what the arg does, there is a doc for that. And if user have put an arg they know what it does already.

Reply:
To be fair, this is nitpicking, I'm fine with whatever, not like anyone is going to hit that warning anyway, who uses 0.43?

Reply:
Yeah, that's the wording I suggested too.

At the moment what I really just want is some sort of consensus on what (if anything) to suggest to @3v1n0.

Reply:
@3v1n0

Is this PR ever going to switch to "with positional argument"? I would love to merge it someday. :)